packages/core/src/transport/failedSendBeacon.ts

```diff
@@ -0,0 +1,43 @@
+import { isExperimentalFeatureEnabled } from '../domain/configuration'
+import { addMonitoringMessage, monitor } from '../domain/internalMonitoring'
+import { generateUUID, startsWith } from '../tools/utils'
+
+// replaced at build time
+declare const __BUILD_ENV__SDK_VERSION__: string
+
+export const LOCAL_STORAGE_KEY = 'datadog-browser-sdk-failed-send-beacon'
+
+export function startFlushFailedSendBeacons() {
+  if (!isExperimentalFeatureEnabled('lower-batch-size')) return
+
+  setTimeout(monitor(flushFailedSendBeacon))
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding a comment to explain this `setTimeout`?

Reply:
sure